### PR TITLE
Chart config tests

### PR DESCRIPTION
**Pull request by jwildfire**
PR to finalize chart-config objects separately from base unit tests. cc @jroden and @samussiah

### CODE REVIEW

**Comment by jwildfire** [comment]
@jroden Coming along well! There's some clean-up to do still.  You can make commits straight to this branch. 

- convert all .js data files to .json like [this one](https://github.com/RhoInc/Webcharts/blob/chart-config-tests/test/samples/chart-config/testConfig.json)
- make sure all json is valid (property names are double quoted etc). you can check [here](https://jsonformatter.org/)
- update `testSettingsList.json` as needed once json conversions are complete. 
- move `./chart-config/chart_gen.py` to the scripts folder
- delete any obsolete files from the `./chart-config/scripts` folder and update the readme as needed to document the process you used for chart generation. 
- make sure that all of the configuration objects you've create render as expected in `./test/visual-tests/`

Finally, remember to review the new testing [wiki page](https://github.com/RhoInc/Webcharts/wiki/testing) and suggest updates as needed

**Comment by jwildfire** [comment]
@samussiah - Reworked the automated config testing. 13 out of 700+ failing right now (mostly due to log axes with 0 values i think). Take a look and see what you think.

**Comment by jwildfire** [comment]
@samussiah - Fixed up the broken tests. ready for your review.

File: package.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
     "minify": "npm run minifyjs && npm run minifycss",
     "minifycss": "cleancss -o css/webcharts.min.css css/webcharts.css",
     "minifyjs": "uglifyjs build/webcharts.js > build/webcharts.min.js",
-    "test": "mocha --require babel-register --recursive ./test/**/*.js",
+    "test": "mocha --timeout 5000 --require babel-register --recursive ./test/**/*.js",
```
Comment (samussiah) [inline package.json]: Does this prevent any single test file from running longer than 5 seconds?

Comment (jwildfire) [inline package.json]: Yes - tests fail if they take longer than 5 seconds (default was 2 seconds).

**Comment by samussiah** [comment]
Awesome work @jwildfire and @jroden !  The test suite is looking substantial, and I love that we're hitting it from so many angles.

**Pull request merged**